packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -1237,10 +1244,6 @@ class ReactDOMServerRenderer {
       }
     }
 
-    if (__DEV__) {
-      validatePropertiesInDevelopment(tag, props);
-    }
```

Comment:
I had to move this before props updates, otherwise warnings never fire for Symbols server-side. I still need to figure out the ramifications of this change. Looks like `defaultValue` loses some SSR warnings, even though all tests pass.